drivers/sensor/sensirion/sdp3x/sdp3x.c

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2024 Yann Gaillard
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT sensirion_sdp3x
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/sys/__assert.h>
```

Comment:
Does not seem to be used

Reply:
I agree for `kernel.h`and `asssert.h`. But remaining are necessary